sdk/keyvault/keyvault/CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ target_link_libraries(az_keyvault PRIVATE az_core_internal)
 add_library (az::keyvault ALIAS az_keyvault)
 
 add_executable (az_keyvault_test test/main.c)
-target_link_libraries(az_keyvault_test PRIVATE az_keyvault az_core_internal)
+target_link_libraries(az_keyvault_test PUBLIC az_keyvault az_core_internal)
```

Comment:
I think we have az_core now; not az_core_internal

Reply:
My understanding is that we still need this - core is public headers, and you need core_internal whenever you need to use internal headers in your code.